Encryption key info update (#8910)
mendix/docs: content/en/docs/appstore/use-content/platform-supported-content/modules/encryption.md

### Limitations

Currently, only AES is supported.

### Configuration {#configuration}

#### EncryptionKey Constant

> Set the `EncryptionKey` constant in the **Private - String en/de-cryption** folder. In module versions 2.2.0 and above, the `EncryptionKey` constant must be 32 characters long, corresponding to the updated 256 bits key length.

Reviewer: I think the stuff about legacy encryption keys should go with the 2.2.0 and above information?
```suggestion
Set the `EncryptionKey` constant in the **Private - String en/de-cryption** folder. In module versions 2.2.0 and above, the `EncryptionKey` constant must be 32 characters long, corresponding to the updated 256 bits key length. You can use the `LegacyEncryptionKey` constant to decrypt strings which were encrypted with 128 bit keys using an older version of the Encryption module.

```

For versions below 2.2.0, ensure the key is 16 characters long to support the 128 bits key length.

> For local application runs, set the key in the **Default value** field. Additionally, the `LegacyEncryptionKey` constant supports 128 bits decryption of strings encrypted using an older version of the Encryption module.

Reviewer: I think it means this - but let me know if I'm wrong.
```suggestion
When running the application locally, set the key as the **Default value** of the `EncryptionKey` constant.
```

#### EncryptionPrefix Constant

Set the `EncryptionPrefix` constant in the **Private - String en/de-cryption** folder. The value depends on the module version you are using:

> * For version 2.2.0 or above, set the constant to `{AES3}`

Reviewer: ```suggestion
* For version 2.2.0 and above, set the constant to `{AES3}`
```

### Limitations

Currently, only AES is supported.

### Configuration {#configuration}

#### EncryptionKey Constant

> Set the `EncryptionKey` constant in the **Private - String en/de-cryption** folder. In module versions 2.2.0 and above, the `EncryptionKey` constant must be 32 characters long, corresponding to the updated 256 bits key length. You can use the `LegacyEncryptionKey` constant to decrypt strings which were encrypted with 128 bit keys using an older version of the Encryption module.

Author: Hi @atennapel, is this text line still applicable here or do I need to remove it - Set the `EncryptionKey` constant in the **Private - String en/de-cryption** folder.

Reviewer 2: The line is correct, but it could be confusing. Perhaps "Set the EncryptionKey constant in your application, which can be found in the Private - String en/de-cryption folder." is clearer.

Author: Done in the commit:https://github.com/mendix/docs/pull/8910/commits/efb483faf81d7dfc07dd0c2ffc4a99aeb4de3de8

For versions below 2.2.0, ensure the key is 16 characters long to support the 128 bits key length.

> When running the application locally, set the key as the **Default value** of the `EncryptionKey` constant.

Reviewer 2: The key should be set as the value for the constant in a configuration (App settings -> configurations).

Author: > The key should be set as the value for the constant in a configuration (App settings -> configurations).

Done in the commit:https://github.com/mendix/docs/pull/8910/commits/936b7fd1f204cb48a4c32da0eb5c5e60fec1ef06
